### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by ericgaspar**
- Upgrade to 4.10.0

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

File: conf/config.js [hunk 1]
```diff
@@ -118,15 +118,50 @@ module.exports = {
...
+     *  sent from users via the encrypted forms on the /support/ page
+     *
+     *  To enable this functionality:
+     *    run `node ./scripts/generate-admin-keys.js`
+     *    save the public key in your config in the value below
+     *    add the private key via the admin panel
+     *    and back it up in a secure manner
+     *
+     */
+    supportMailboxPublicKey: '',
```
Comment (frju365) [inline conf/config.js:136]: I was thinking if we must add this option as default ?

Comment (ericgaspar) [reply conf/config.js:136]: I think that this option doesn't do anything unless a key value is set.
I added a bunch of settings that disappeared from the config file in the admin parte such as: `adminEmail`, `removeDonateButton`, `blockDailyCheck`. I don't know why those options got removed from the config.js example file.

**Review by alexAubin — APPROVED** [review]
Naively LGTM

**Pull request merged**